index.ts

```diff
@@ -1094,6 +1102,35 @@ commander
     });
   });
 
+/**
+ * Clasp run <functionName>
+ * This function runs your script in the cloud. You must supply
+ * the functionName params. For now, it can 
+ * only run functions that do not require other authorization.
+ * @param functionName function in the script that you want to run
+ * @see https://developers.google.com/apps-script/api/reference/rest/v1/scripts/run
+ * Note: to use this command, you must have used `clasp login --ownkey`
+ */
+commander
+  .command('run <functionName>')
+  .description('Run a function in your Apps Scripts project')
+  .action((functionName) => {
+    getAPICredentials(true, async () => {
+      await checkIfOnline();
+      getProjectSettings().then(({ scriptId }: ProjectSettings) => {
+        const params = {
+          'scriptId': scriptId,
+          'function': functionName,
+          'devMode': true
+        };
```

Comment:
Minimize `params` (no quotes in keys, make key/value the same if possible).

Reply:
Got rid of the quotes and of `scriptId`. However, it still appeared to need both `function` and `devMode`